py/torch_tensorrt/dynamo/conversion/ops_evaluators.py

```diff
@@ -23,6 +25,7 @@ def getitem_validator(getitem_node: Node) -> bool:
 
 # TODO: Subsequent evaluators should be registered here with their own validators
 @dynamo_tensorrt_converter(operator.getitem, capability_validator=getitem_validator)
+@dynamo_tensorrt_converter(builtins.getattr)
```

Comment:
What does the `getattr` builtin do in the FX graph? Are there any sample uses of it in models? It may need a validator like `operator.getitem`

Reply:
Thanks for the comment! Yes, `builtins.getattr` is used in the test here: 
```python
class Arange(nn.Module):
    def forward(self, x):
        return torch.ops.aten.arange.start_step(start, x.shape[0], step)
```
where `x.shape` calls `builtins.getattr`, as the graph below: 
```
.INFO:harness:FX graph= graph():
    %x : [num_users=1] = placeholder[target=x]
    %getattr_1 : [num_users=1] = call_function[target=builtins.getattr](args = (%x, shape), kwargs = {})
    %getitem : [num_users=1] = call_function[target=operator.getitem](args = (%getattr_1, 0), kwargs = {})
    %arange_start_step : [num_users=1] = call_function[target=torch.ops.aten.arange.start_step](args = (3, %getitem, 3), kwargs = {})
    return arange_start_step
```

I thought about adding `validator`, but it depends on the attributes that the caller has. It seems not general to only allow `shape` attr I think? Do you have any idea about validator?

Reply:
I don't think `builtins.getattr` is a standard op, and should not appear in Dynamo graphs after ATen tracing. Could you try running the test case with `use_dynamo_tracer=True`?

Reply:
Cool, it disappeared, thanks!